tests/e2e/performance/test_small_file_workload.py

```diff
@@ -353,104 +462,82 @@ def setup(self):
         ],
     )
     @pytest.mark.polarion_id("OCS-1295")
-    def test_smallfile_workload(
-        self, ripsaw, es, file_size, files, threads, samples, interface
-    ):
+    def test_smallfile_workload(self, file_size, files, threads, samples, interface):
```

Comment:
Doc string required for params in function test_smallfile_workload()

Reply:
done